### DISCUSSION TITLE
Slightly Wrong Translation

### DISCUSSION THREAD

**Discussion by BananeDollar** [Ludeon/RimWorld-de#82] [General]
Hey. I found a wrong translation.

"Mentaler Band" should be "Mentales Band".
The Ending in "er" should be "s"

Yea German is strange :D

![Mentaler Band](https://github.com/Ludeon/RimWorld-de/assets/22256432/b37b9b53-1872-4a3f-9cbe-c1e23da5f680)

**Comment by Ragnar-F** [comment]
Thanks for reporting. Fixed with e3a6160.
